java/src/jmri/InstanceManager.java

```diff
@@ -555,16 +555,12 @@ static public ThrottleManager throttleManagerInstance() {
      *
      *                      Please don't create any more of these
      * ****************************************************************************/
-    // Simplification order - for each type, starting with those not in the jmri package:
-    //   1) Remove it from jmri.managers.DefaultInstanceInitializer, get tests to build & run
-    //   2) Remove the setter from here, get tests to build & run
-    //   3) Remove the accessor from here, get tests to build & run
     /**
      * Deprecated, use @{link #getDefault} directly.
      *
      * @return the default block manager. May not be the only instance. In use
      *         by scripts.
```

Comment:
Shouldn't these two lines be changed to:

`* @return the default block manager. May not be the only instance.`

Reply:
@danielb987 Thanks for catching that. Fixed.